mantisshrimp/models/mantis_rcnn/mantis_mask_rcnn.py

```diff
@@ -5,20 +5,57 @@
 from mantisshrimp.models.mantis_rcnn.rcnn_param_groups import *
 from mantisshrimp.models.mantis_rcnn.mantis_rcnn import *
 from mantisshrimp.models.mantis_rcnn.mantis_faster_rcnn import *
+from mantisshrimp.backbones import *
 
 
 class MantisMaskRCNN(MantisRCNN):
     @delegates(MaskRCNN.__init__)
-    def __init__(self, n_class, h=256, pretrained=True, metrics=None, **kwargs):
+    def __init__(
+        self, n_class: int, backbone: nn.Module = None, metrics=None, **kwargs,
```

Comment:
Can we rename `n_class` to `num_classes`? I think it's clearer and it's what torchvision uses

I'll be doing that for fasterrcnn as well

Reply:
Renamed. For consistenct with torchvision